exasol_integration_test_docker_environment/lib/base/db_os_executor.py

```diff
@@ -87,10 +93,28 @@ def __del__(self):
         self.close()
 
     def exec(self, cmd: str) -> ExecResult:
-        result = self._connection.run(cmd)
+        result = self._connection.run(cmd, warn=True, hide=True)
         output = result.stdout.encode("utf-8")
         return ExecResult(result.exited, output)
 
+    def prepare(self):
+        print("SshExecutor: probing until ssh connection is available")
+        retry = 0
+        while retry < 20:
+            try:
+                retry += 1
+                prefix = "," if retry > 1 else ""
+                print(f'{prefix}{retry}', end='')
```

Comment:
prints are not reliable, because things happen in parallel, it is likely the output is garbage in the end

Reply:
Yes I intentionally left the code to wait for your proposals.
Should I remove the printing?

Reply:
I removed the printing now.